<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdaterWindow</class>
 <widget class="QWidget" name="UpdaterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Crow Translate</string>
  </property>
  <property name="windowIcon">
   <iconset theme="software-update-available">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;A new version of Crow Translate is available! Updates add functionality and improve the stability of the application. Most often.&lt;br/&gt;You can also download the release manually from this &lt;a href=&quot;https://github.com/Shatur95/crow-translate/releases&quot;&gt;link&lt;/a&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextBrowserInteraction</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="versionsLabel"/>
   </item>
   <item>
    <widget class="QTextEdit" name="changelogTextEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="progressLayout">
     <item>
      <widget class="QProgressBar" name="downloadBar"/>
     </item>
     <item>
      <widget class="QPushButton" name="cancelDownloadButton">
       <property name="toolTip">
        <string>Cancel download</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <widget class="QPushButton" name="downloadButton">
       <property name="toolTip">
        <string>Download the installer to the Downloads folder</string>
       </property>
       <property name="text">
        <string>Download</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="installButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Exit the program and run the installer</string>
       </property>
       <property name="text">
        <string>Install</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="updateStatusLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateLaterButton">
       <property name="toolTip">
        <string>Close this window</string>
       </property>
       <property name="text">
        <string>Update later</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>installButton</sender>
   <signal>clicked()</signal>
   <receiver>UpdaterWindow</receiver>
   <slot>install()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>downloadButton</sender>
   <signal>clicked()</signal>
   <receiver>UpdaterWindow</receiver>
   <slot>download()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateLaterButton</sender>
   <signal>clicked()</signal>
   <receiver>UpdaterWindow</receiver>
   <slot>updateLater()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelDownloadButton</sender>
   <signal>clicked()</signal>
   <receiver>UpdaterWindow</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>install()</slot>
  <slot>cancel()</slot>
  <slot>updateLater()</slot>
  <slot>download()</slot>
 </slots>
</ui>
